quickstart/known-limitations.md

```diff
@@ -6,7 +6,7 @@ Once provisioned, neither Mayastor Disk Pools nor Mayastor Volumes can be re-siz
 
 ## Snapshots and Clones
 
-Mayastor has no snapshot or cloning capabilities.
+Mayastor has no cloning capabilities, you can only create a snapshot.
```

Comment:
Mayastor has no cloning capabilities, you can only create a snapshot for a single replica volume.

cc @avishnu @tiagolobocastro Since we decided to keep it, we should include this at all versions.